ui/apps/dashboard/src/app/api/inngest/client.ts

```diff
@@ -0,0 +1,7 @@
+import { realtimeMiddleware } from '@inngest/realtime/middleware';
+import { Inngest } from 'inngest';
+
+export const inngest = new Inngest({
```

Comment:
Which event key is this using? Our UI's Vercel project only has 1 event key right now and it's the public one that has a whitelist, so it won't work

Reply:
This will be using new env keys added as you'd expect.  We can override these.